src/main/java/com/valmiki/adhyapakservice/controller/CourseController.java

```diff
@@ -0,0 +1,33 @@
+package com.valmiki.adhyapakservice.controller;
+import com.valmiki.adhyapakservice.entity.Courses;
+import com.valmiki.adhyapakservice.service.CourseService;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@RequestMapping("/courses")
+public class CourseController {
+
+    private CourseService courseService;
```

Comment:
Use @Autowired here, and then you won't need to create object

Reply:
fixed